csharp/compatibility_tests/v3.0.0/src/Google.Protobuf.Test/CodedInputStreamTest.cs

```diff
@@ -415,6 +415,25 @@ public void TestSlowPathAvoidance()
             }
         }
 
+        [Test]

```

Comment:
Is there any reason you've added this test in here as well? We don't normally do that.